install/turn-things-off.sh

```diff
@@ -6,6 +6,10 @@ if [[ -n "$MINIMIZE_DOWNTIME" ]]; then
 else
   # Clean up old stuff and ensure nothing is working while we install/update
   $dc down -t $STOP_TIMEOUT --rmi local --remove-orphans
+  # TODO(getsentry/self-hosted#2489)
+  if docker volume ls | grep -qw sentry-zookeeper; then
+    docker volume rm sentry-zookeeper
```

Comment:
Well, I love this that said if the upgrade fails for some reason you have now lost data?

Reply:
Moved this removal to the end of the installation then to ensure it will only run if the upgrade succeeds.